LICENSE

```diff
@@ -0,0 +1 @@
+FREE 99
```

Comment:
Hey @nicholaspulido why did you add this line?

Reply:
How else will people know what license this sweet code is under?